pkg/metrics/nextgengrafana/tidb_summary_with_keyspace_name.jsonnet

```diff
@@ -386,7 +446,10 @@ local rightPanelPos = {x:panelW, y:0, w:panelW, h:panelH};
 newDash
 .addPanel(
   serverRow
-  .addPanel(connectionP, gridPos=leftPanelPos)
+  .addPanel(uptimeP, gridPos=leftPanelPos)
```

Comment:
Should these panels be generated into tidb_summary_with_keyspace_name.json? Why does it feel like they were generated into the tidb_with_keyspace_name.json file in this PR?

Reply:
Yeah, regenerated the tidb_summary_with_keyspace_name.json.